test/framework/gardener_utils.go

```diff
@@ -576,3 +576,78 @@ func ScaleGardenerScheduler(setupContextTimeout time.Duration, client client.Cli
 func ScaleGardenerControllerManager(setupContextTimeout time.Duration, client client.Client, desiredReplicas *int32) (*int32, error) {
 	return ScaleDeployment(setupContextTimeout, client, desiredReplicas, "gardener-controller-manager", gardencorev1beta1constants.GardenNamespace)
 }
+
+// RegisterSeed Creates a seed from a seed Object and waits until it is successfully reconciled
+func (f *GardenerFramework) RegisterSeed(ctx context.Context, seed *gardencorev1beta1.Seed) error {
```

Comment:
maybe `CreateSeed` is more readable as you create a seed and also at the end `WaitForSeedToBeCreated`.
Register would make sense if you register a shoot as seed but this function only creates a seed